### ISSUE TITLE
width="NaN" in generated code for fixed height amp-iframe

### ISSUE THREAD

**Issue by pbakaus** [ampproject/amphtml#4377]
So this is just odd, and doesn't seem to cause visible issues atm, but thought I'd be good to report nonetheless:

On a page like https://paulbakaus.com/tutorials/css/flexbox-freebie-auto-growing-list-for-amp-roadmap/ that has iframes with fixed height, like so:

```html
<amp-iframe src="https://gfycat.com/ifr/PlumpLivelyJunebug" height="400" frameborder="0" allowfullscreen="" sandbox="allow-scripts allow-same-origin" layout="fixed-height"><div placeholder="" class="amp-wp-iframe-placeholder"></div></amp-iframe>
```

the generated code (after AMP runs) looks like this:

```html
<amp-iframe src="https://gfycat.com/ifr/PlumpLivelyJunebug" height="400" frameborder="0" allowfullscreen="" sandbox="allow-scripts allow-same-origin" layout="fixed-height" class="-amp-element -amp-layout-fixed-height -amp-layout-size-defined -amp-layout" id="AMP_2" style="height: 400px;">
  <div placeholder="" class="amp-wp-iframe-placeholder amp-hidden"></div>
  <i-amp-scroll-container class="amp-active">
    <iframe class="-amp-fill-content" width="NaN" height="400" name="amp_iframe0" frameborder="0" allowfullscreen="" sandbox="allow-scripts allow-same-origin" src="https://gfycat.com/ifr/PlumpLivelyJunebug" style="z-index: 0;"></iframe>
  </i-amp-scroll-container>
</amp-iframe>
```

Notice the width="NaN" in the actual `<iframe>`?

I freak out whenever I see a "NaN" in my code, so I thought you might want to take a look at this :)

**Comment by dvoytenko** [comment]
Yep. Just needs to be fixed. /to @muxin

**Comment by zhouyx** [comment]
Is this related? @dvoytenko #3968

**Comment by dvoytenko** [comment]
Yep. Oops. LGTM

**Comment by zhouyx** [comment]
should be fixed by #3968
